packages/commuter-server/flow-typed/npm/aws-sdk_vx.x.x.js

```diff
@@ -8,1396 +8,1452 @@
  *
  * Fill this stub out by replacing all the `any` types.
  *
- * Once filled out, we encourage you to share your work with the 
- * community by sending a pull request to: 
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
  * https://github.com/flowtype/flow-typed
  */
 
-declare module 'aws-sdk' {
-  declare module.exports: any;
-}
-
-/**
+declare type httpAgent = Object;
+declare type Credentials = any;
+
+declare type CredentialProviderChain = any;
+
+declare type $ConfigOptions = {
+  accessKeyId?: string,
+  secretAccessKey?: string,
+  sessionToken?: Credentials,
+  credentials?: Credentials,
+  credentialProvider?: CredentialProviderChain,
+  region?: string,
+  maxRetries?: number,
+  maxRedirects?: number,
+  sslEnabled?: boolean,
+  paramValidation?:
+    | boolean
+    | {
+        min?: boolean,
+        max?: boolean,
+        pattern?: boolean,
+        enum?: boolean
+      },
+  computeChecksums?: boolean,
+  convertResponseTypes?: boolean,
+  correctClockSkew?: boolean,
+  s3ForcePathStyle?: boolean,
+  s3BucketEndpoint?: boolean,
+  s3DisableBodySigning?: boolean,
+  retryDelayOptions?: {
+    base?: number,
+    customBackoff?: (retryCount: number) => number
+  },
+  httpOptions?: {
+    proxy?: string,
+    agent?: httpAgent,
+    timeout?: number,
+    xhrAsync?: boolean,
+    xhrWithCredentials?: boolean
+  },
+  apiVersion?: string | Date,
+  apiVersions?: { [key: string]: string | Date },
+  logger?: void | { write: Function } | { log: Function },
+  systemClockOffset?: number,
+  signatureVersion?: string,
+  signatureCache?: boolean
+};
```

Comment:
I brought in definitions from https://github.com/motiz88/aws-sdk-flow-decls, only for S3 though (since that's all we use here). Otherwise this is a mighty hefty file.